Source/Noesis.Javascript/JavascriptContext.cpp

```diff
@@ -502,8 +516,18 @@ JavascriptContext::GetObjectWrapperTemplate()
 void
 JavascriptContext::RegisterFunction(System::Object^ f)
 {
-    // Note that while we do store WeakReferences, we never clean up this hashtable,
-    // so it will just grow and grow.
+    if (mFunctions->Count > 100)

```

Comment:
this will do the check each time as soon there are 100+ *live* functions. Maybe something like `count % 100 = 0` would be more suitable? But well, I guess it would not even matter to do the check each time. Just wondering about the arbitrary treshold of 100 here.

Reply:
Yeah, it's a bit arbitrary, but every periodic cleanup threshold will be...

Reply:
Well but this is not periodic for 100+ live functions :)